quakemigrate/signal/local_mag/amplitude.py

```diff
@@ -262,7 +262,10 @@ def get_amplitudes(self, event, lut):
 
         # Loop through stations, calculating amplitude info
         for i, station_data in lut.station_data.iterrows():
-            station = station_data["Name"]
+            network = station_data["Network"]
+            station = station_data["Station"]
+            location = station_data["Location"]
+            channel = station_data["Channel"]
```

Comment:
Is there any need to unpack here, or can we just have `station_data.Network` when `network` is needed, etc.